src/main/scala/de/upb/cs/swt/delphi/instanceregistry/io/swagger/client/model/Event.scala

```diff
@@ -121,31 +136,31 @@ object RegistryEventFactory {
     * @return RegistryEvent with the respective type and payload.
     */
   def createNumbersChangedEvent(componentType: ComponentType, newNumber: Int) : RegistryEvent =
-    RegistryEvent(EventType.NumbersChangedEvent, NumbersChangedPayload(componentType, newNumber))
+    RegistryEvent(EventType.NumbersChangedEvent, NumbersChangedPayload(componentType, newNumber),DateTime.now)
```

Comment:
Please add a space after the comma. This also holds for the other occurrences below.